src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1882,6 +1882,31 @@ public void testContainerWithMoreCpuOptions() throws Exception {
   }
 
   @Test
+  public void testContainerWithNanoCpusOptions() throws Exception {
```

Comment:
Can you please explain why it should be done in this case? If I understand this correctly, I'd have to mock json response and then parse it to HostConfig, which seems kinda confusing and I'm not really sure how to do it correctly. Testing it against docker daemon, like the rest of HostConfig properties, seems like a better choice.